crates/next-core/src/next_font_google/mod.rs

```diff
@@ -82,14 +82,19 @@ impl ImportMappingReplacement for NextFontGoogleReplacer {
                     formatdoc!(
                         r#"
                             import cssModule from "@vercel/turbopack-next/internal/font/google/cssmodule.module.css?{}";
-                            export default {{
+                            const fontData = {{
                                 className: cssModule.className,
                                 style: {{
                                     fontFamily: "{}",
                                     {}{}
                                 }},
-                                variable: cssModule.variable
                             }};
+
+                            if (cssModule.variable != null) {{
+                                fontData.variable = cssModule.variable;
+                            }}
```

Comment:
Why is this change included?

Reply:
It's unrelated, but a small issue issue I fixed while adding the new test. Could have been a new PR I suppose.